jekyll/_cci2/server/air-gapped-installation/phase-4-configuring-nomad-clients.adoc

```diff
@@ -0,0 +1,47 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 4 - Configuring Nomad Clients
```

Comment:
I don't know if we want to tell them how to install nomad.  I wonder if we should point them to the nomad docs instead.  Then we have less risk of falling out of date with their process and version numbers